website/src/components/demo-launcher.js

```diff
@@ -1,34 +1,14 @@
 import React, {Component} from 'react';
 import {connect} from 'react-redux';
 import autobind from 'autobind-decorator';
+import {FlyToInterpolator} from 'react-map-gl';
```

Comment:
Why not use deck.gl's version?

Reply:
The website examples interaction is using the react-map-gl controller.